githttp.go

```diff
@@ -10,6 +10,14 @@ import (
 	"strings"
 )
 
+type ResolveInfo struct {
+	Path    string
+	Request *http.Request
```

Comment:
golint would warn that all 3 of these things are exported and should have comment or be unexported.

Is `Path` the same as `Request.URL.Path`? If so, are both needed?